ckanext/switzerland/templates/package/read_base.html

```diff
@@ -14,3 +17,11 @@
 
 {% block primary_content %}
 {% endblock %}
+{%  block body_extras %}$
+  {{ super() }}
+  <script type="text/javascript">
+      customDimensionVisit = {};
+      customDimensionVisit['dimension'+'{{ piwik_config.custom_dimension_visit_dataset_id or 0 }}'] = '{{ pkg.organization.name }}';
+      _paq.push(['trackPageView', ' {{ pkg.name }}', customDimensionVisit]);
```

Comment:
There is an extra space between `'` and `{{ pkg.name }}`. Is this needed?

Reply:
no

Reply:
ok ;)